docs/using/libraries/ruby/index.rst

```diff
@@ -113,6 +114,11 @@ Likewise, lists constructed in Polar may be passed into Ruby methods:
   user = User.new(["HR", "payroll"])
   raise "should be allowed" unless oso.allowed?(user, "foo", "bar")
 
+There is currently no syntax for random access to a list element within a policy;
+i.e., there is no Polar equivalent of the Ruby expression ``user.groups[1]``.
+To access the elements of a list, you may iterate over it with :ref:`operator-in`
+or destructure it with :ref:`pattern matching <patterns-and-matching>`.
```

Comment:
Might be worth mentioning that you could also access it as `user.groups.at(1)`?

Reply:
This is a general comment that we're saying "there's no syntax for index access, but you can iterate over all elements of a list" instead of ", ...but you can use List methods defined in the host language to access an element at a particular position," which seems much more analogous to index access than does enumerating with `in`.

Applies to all language library docs -- not just Ruby.

Reply:
bd8bea72